holoviews/util/__init__.py

```diff
@@ -502,15 +502,22 @@ def __call__(self, *args, **options):
                 if k not in Store.output_settings.allowed:
                     raise KeyError('Invalid keyword: %s' % k)
             if 'filename' in options:
-                def save_fn(obj, renderer): renderer.save(obj, options['filename'])
-                Store.output_settings.output(line=line, cell=obj, cell_runner=save_fn,
-                                             help_prompt=help_prompt, **options)
-            elif warn:
-                self.warning("hv.output not supplied a filename to export the "
-                             "given object. This call will have no effect." )
-            return obj
-        elif obj is not None:
-            return obj
+                if util.config.future_deprecations:
```

Comment:
If it's a future deprecation you can't remove the actual code to save the output now.

Reply:
The `filename` argument still works via the display code path.

Reply:
Ah right, it's now calling the actual codepath it would use for the cell magic.